README.md

```diff
@@ -5,20 +5,66 @@ the [HIP][] programming language and optimized for AMD's latest discrete GPUs.
 ## rocBLAS Wiki
 The [wiki][] has helpful information about building the rocblas library, samples and tests.
 
+## Building rocBLAS
+The build infrastructure for rocBLAS is based on [Cmake](https://cmake.org/) v1.5.  This is the version of cmake available on ROCm supported platforms.  Examples of installing cmake:
+* Ubuntu: `sudo apt install cmake-qt-gui`
+* Fedora: `sudo dnf install cmake-gui`
+
+### Library
+The rocBLAS library has one dependency named [Tensile](https://github.com/ROCmSoftwarePlatform/Tensile), which supplies the high-performance implementation of xGEMM.  Tensile is downloaded by cmake during library configuration and automatically configured as part of the build, so no further action is required by the user to set it up.  Tensile itself is predominately written in python, so it does bring python dependencies which can easily be installed with distro package managers.  The rocBLAS library contains both host and device code, so the HCC compiler must be specified during cmake configuration to properly initialize build tools.  Example steps to build rocblas:
+
+#### (One time only)
+* Ubuntu: `sudo apt install python2.7 python-yaml`
+* Fedora: `sudo dnf install python PyYAML`
+
+#### Configure and build steps
+1.  `mkdir -p [ROCBLAS_BUILD_DIR]/release`
```

Comment:
For compatibility with other rocm packages should this be    build   in place of   release?